src/BaselineOfSpec/BaselineOfSpec.class.st

```diff
@@ -7,21 +7,6 @@ Class {
 	#category : #BaselineOfSpec
 }
 
-{ #category : #'pre/post load' }
-BaselineOfSpec >> addHackToLoadSpec [
```

Comment:
Removing the hacks will work for now because there is only few changes between Pharo8 and Spec version, but if at some point we create a change that will make a package dirty *in the middle* of an update, it will crash once again. 

I don't think it should be removed just like that without any solution. The case I described before can just be achieved by moving a class to another package. So it is not uncommon.

Reply:
That's why I did not integrate it blindly and made a PR for this one :).
Before integrating this one I want to have a green CI.

Already loading in Pharo7 should give an scenario like the one you say, no?